orchestra/tests/helpers/iterations.py

```diff
@@ -92,14 +90,13 @@ def _verify_iteration_data(test, iterations):
         siblings = assignment.iterations.order_by('start_datetime')
         if iteration.status == Iteration.Status.PROCESSING:
             # Iterations should not have data until submitted
-            test.assertEqual(iteration.submitted_data, {})
+            assert iteration.submitted_data == {}
```

Comment:
using assert instead of self.assertEqual won't print out what the expected/actual values are:
```
    assert b == c
AssertionError
```
vs.
```
    self.assertEqual(b, c)
AssertionError: 1 != 2
```
Can we add the functions to the base test case?